mitocw/edx-platform
# Fixed progress page update on save ccx

### Background
fixes https://github.com/mitocw/edx-platform/issues/252

### What is done in this PR
**Studio Updates:** None.
**LMS Updates:** 
 - Add signal on save_ccx api, So that other application can be up to date when ccx contents change,

@pdpinch @giocalitri 

### To test:
- Create a coach
- create a student
- Login to edX platform as coach, open coach dashboard. change schedule of subsection aka sequential and press save.
<img width="615" alt="screen shot 2016-07-14 at 7 32 14 pm" src="https://cloud.githubusercontent.com/assets/10431250/16843215/cd27feea-49f9-11e6-9326-56e91ae61a5a.png">
<img width="757" alt="screen shot 2016-07-14 at 7 32 23 pm" src="https://cloud.githubusercontent.com/assets/10431250/16843216/cd2b91fe-49f9-11e6-83bf-ec95ef18c708.png">

- Login as student (CCX enrolled student). Go to dashboard then open ccx course and go to progress and see the due date of that subsection(problem).
<img width="1036" alt="screen shot 2016-07-14 at 7 34 41 pm" src="https://cloud.githubusercontent.com/assets/10431250/16843404/8c3e5a0e-49fa-11e6-8bbc-30b4c59773cd.png">
<img width="971" alt="screen shot 2016-07-14 at 7 34 52 pm" src="https://cloud.githubusercontent.com/assets/10431250/16843408/8cfca5ae-49fa-11e6-8024-c25fad06b69a.png">
<img width="653" alt="screen shot 2016-07-14 at 7 37 25 pm" src="https://cloud.githubusercontent.com/assets/10431250/16843409/8d27ef48-49fa-11e6-85f9-5c672c6f6250.png">
